cmd/lakectl/cmd/branch_revert.go

```diff
@@ -12,6 +12,7 @@ import (
 const (
 	branchRevertCmdArgs  = 2
 	ParentNumberFlagName = "parent-number"
+	allowEmptyRevert     = "allow-empty-revert"
```

Comment:
allow-empty-commit?

There are two situations where we might want to allow empty commits: during regular commits and during reverts. I propose that we keep it the same.

What do you think?